packages/web-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -26,6 +27,10 @@ const _Checkbox = (props: SpiritCheckboxProps, ref: ForwardedRef<HTMLInputElemen
   } = modifiedProps;
   const { styleProps, props: otherProps } = useStyleProps(restProps);
   const [ids, register] = useAriaIds(ariaDescribedBy);
+  const role = useValidationTextRole({
```

Comment:
maybe the role variable should have a better name? So I know it is related only to the validationText.

```suggestion
  const validationTextRole = useValidationTextRole({
```

Reply:
I would be ok if we leave it, but maybe it can be slightly better :)